db/schema.rb

```diff
@@ -82,9 +82,9 @@
     t.string "national_insurance_number", limit: 9
     t.string "email_address", limit: 256
     t.boolean "mostly_teaching_eligible_subjects"
+    t.datetime "submitted_at"
```

Comment:
This has happened again, can you edit your commit to remove this file?

Reply:
I think I've sorted it now